CHANGES.md

```diff
@@ -16,14 +16,43 @@ Note that it is not required to have an entry for every pull request.
 Instead, please try to add only changes that are meaningful to users who read
 this file to understand what changed.
 
-## v-next (post-v0.2.0)
+## v-next (post-v0.2.\*)
 
 - educational materials
 - new features
 - breaking changes
 - bug fixes
 - other
 
+## v0.2.1
+
+Update to `raiwidgets` package.
+
+- educational materials:
+  - add imports for `MimicExplainer` and `PFIExplainer`
+- features
+
+  - migrate cohort and dataset utilities to core-ui
+  - remove circular dependencies
+  - convert flask comms class to function
+  - add confirmation popup in `ExplanationDashboard` when cancelling editing
+    a cohort.
+  - fix clipped dropdown in `FairnessDashboard`
+  - keep `FairnessDashboard` chart state in single model view
+  - in large data view, change `ErrorAnalysisDashboard` cohort state to use
+    full data instead of downsampled explanation data
+  - updates to `ErrorAnalysisDashboard`:
+    - add true y and predicted y values
+    - fix categorical labels handling in what if panel
+    - add breadcrumb to error explorer view and minor style adjustments to
+      navigation panel
+    - fix indexing issues for all data cohort
+    - add color column to local explanation view
+
+- pypi:
+
+  - raiwidgets: release v0.2.1
```

Comment:
What's this for? It already says 0.2.1 at the top.

Also, if this was a minor release (e.g., 0.3.0) then we'd need to increase the version in the npm packages. But it isn't, so just FYI

Reply:
done, removed!